.scalafix.conf

```diff
@@ -0,0 +1,7 @@
+OrganizeImports {
```

Comment:
I wonder if this is the same as the following:
```
OrganizeImports.preset = INTELLIJ_2020_3
```

See more here:
https://scalacenter.github.io/scalafix/docs/rules/OrganizeImports.html#intellij_2020_3

Reply:
there's some differences in imports ordering, but IMO its minor thing. I set
```
OrganizeImports.preset = INTELLIJ_2020_3
```
as you have proposed. Thanks